dashboard/src/actions/tableOfContentActions.js

```diff
@@ -18,7 +20,8 @@ export const fetchTOC =
         });
       }
     } catch (error) {
-      return error;
+      const msg = error.response?.data?.message;
+      dispatch(showToast(DANGER, msg ? msg : `Error response: ERROR_MSG`));
```

Comment:
You're not wrong, but I just copied this in the process of debugging my refactoring so it wouldn't just hang without feedback on an error as it had been. I didn't really think about it all that much. I'm not sure if Javascript has the standard status messages we're used to in `requests.Response`, but that'd probably be a decent fallback instead of just `ERROR_MSG` ... which is kinda weird.